utils/catalog-parser/src/test/java/de/sovity/edc/utils/catalog/mapper/DspContractOfferUtilsTest.java

```diff
@@ -2,11 +2,13 @@
 
 import jakarta.json.Json;
 import lombok.val;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
 import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
 
 class DspContractOfferUtilsTest {
+    @Disabled("hotfix: the stable ID or the way the new ID is build prevents a contract negotiation.")
```

Comment:
I think it's fine to remove this test, if we have / add a dedicated test for the stable id stuff. We could then use the Stable ID Code from the Broker

Reply:
ok